app.js

```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
+    alert('I play video games just about everyday!');
+  }else{
+    alert("Your response wasn't yes or no!");
+  }
 
 var juice = prompt('Do I like grapejuice? (answer yes or no)');
-if (juice.trim().toLowerCase() == 'yes' || juice.trim().toLowerCase() == 'y' ){
-  alert('Bleh, I hate it!');
-}
-else if (juice.trim().toLowerCase() == 'no' || juice.trim().toLowerCase() == 'n' ){
-  alert('You are correct!')
-}
-else{
-  alert("Your resonse wasn't yes or no!");
-}
+
+  if (juice == null) {
+    alert('Canceled');
+  }else if (juice.trim().toLowerCase() == 'yes' || juice.trim().toLowerCase() == 'y' ){
+    alert('Bleh, I hate it!');
+  }else if (juice.trim().toLowerCase() == 'no' || juice.trim().toLowerCase() == 'n' ){
+    alert('You are correct!')
+  }else{
+    alert("Your resonse wasn't yes or no!");
+  }
 
 var travel = prompt("Do you think that I've ever lived abroad? (answer yes or no)");
-if (travel.trim().toLowerCase() == 'yes' || travel.trim().toLowerCase() == 'y' ){
-  alert('You are correct!');
-}
-else if (juice.trim().toLowerCase() == 'no' || juice.trim().toLowerCase() == 'n' ){
-  alert('I lived in Italy and Afghanistan!');
-}
-else{
-  alert("Your response wasn't yes or no!");
-}
+
+  if (travel == null) {
+    alert('Canceled');
+  }else if (travel.trim().toLowerCase() == 'yes' || travel.trim().toLowerCase() == 'y' ){
+    alert('You are correct!');
+  }else if (travel.trim().toLowerCase() == 'no' || travel.trim().toLowerCase() == 'n' ){
+    alert('I lived in Italy and Afghanistan!');
+  }else{
+    alert("Your response wasn't yes or no!");
+  }
 
 var animals = prompt("Am I an animal lover? (answer yes or no)");
-if (animals.trim().toLowerCase() == 'yes' || animals.trim().toLowerCase() == 'y' ){
+
+if (animals == null) {
+  alert('Canceled');
+}else if (animals.trim().toLowerCase() == 'yes' || animals.trim().toLowerCase() == 'y' ){
   alert('You are correct!');
-}
-else if (animals.trim().toLowerCase() == 'no' || animals.trim().toLowerCase() == 'n' ){
+}else if (animals.trim().toLowerCase() == 'no' || animals.trim().toLowerCase() == 'n' ){
   alert('But I LOVE animals!');
-}
-else{
+}else{
   alert("Your response wasn't yes or no!");
 }
 
 var people = prompt('Do I like people? (answer yes or no)');
-if (people.trim().toLowerCase() == 'yes' || people.trim().toLowerCase() == 'y' ){
+
+if (people == null) {
+  alert('Canceled');
+}else if (people.trim().toLowerCase() == 'yes' || people.trim().toLowerCase() == 'y' ){
   alert('You are correct!');
-}
-else if (people.trim().toLowerCase() == 'no' || people.trim().toLowerCase() == 'n' ){
+}else if (people.trim().toLowerCase() == 'no' || people.trim().toLowerCase() == 'n' ){
   alert('What kind of person do you think I am!?');
-}
-else{
+}else{
   alert("Your response wasn't yes or no!");
 }
+
+var number = 7;
+var done = true;
+var rightNum;
+for (var num = 4; num > 0 && done; num--){
+  var rightNum = prompt('What is my favorite number?');
```

Comment:
You've already declared this variable above the for loop.
Also, rightNum implies that the guess is the right number. You want to show that this isn't the answer but the user imput with the name.